### ISSUE TITLE
the ability to hide tabs and extend the content

### ISSUE THREAD

**Issue by calidion** [ionic-team/ionic#9883]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[x ] **2.x**

**I'm submitting a ...**  (check one with "x")
[ ] bug report
[x ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
1、seem tabs cannot be hidden  by api
2、 ion-content cannot be extended if tabs are manually hidden, ion-tool-bar will be given a 50px bottom.

**Expected behavior:**
1、 tabs can be hidden  by api
2、 ion-content can be extended if tabs are manually hidden, ion-tool-bar will not be given a 50px bottom when tabs are hidden.

**Comment by jgw96** [comment]
Hello, thanks for using Ionic! Would you mind explaining your issue a little more? How are you hiding the tabs?

**Comment by AndreasGassmann** [comment]
I had a similar issue when I was trying to hide tabs on a single sub page (so I couldn't use the hideTabsOnSubPages option).

I tried all possible solutions available at the time: https://forum.ionicframework.com/t/ionic2-hide-tabs/37998

When using `document.querySelectorAll('.tabbar');` and hiding the tabs, the issue was that the tabs were still "blocking the space", even though they were not visible anymore. So if you added a footer, it would be placed above the invisible tabs. After a while I stopped trying and ended up using a modal instead. Maybe this solution could work though: https://gist.github.com/MT--/505a1c385f08fa1b0d6f6f3648ddf582

But I agree with @calidion that it would be nice to have a simple API to hide tabs on a specific sub page.
